doc/_templates/autosummary/module.rst

```diff
@@ -0,0 +1,7 @@
+{{ fullname }}
+{{ underline }}
+
+.. automodule:: {{fullname}}
```

Comment:
This is to customize autosummary to render modules, or it will be in a bad shape for submodules.